src/Components/DomainList/DomainList.tsx

```diff
@@ -203,6 +188,8 @@ export const DomainList = () => {
 
   // Note that we perform the sort as part of the component's render logic and not in onSort.
   // We shouldn't store the list of data in state because we don't want to have to sync that with props.
+  console.debug('Domainlist:domains');
+  console.debug(domains);
```

Comment:
We used this in debugging. Did they stay intentionally or are leftovers?

Reply:
thanks! it is leftover!